packages/account/src/Sections/Profile/LanguageSettings/index.js

```diff
@@ -0,0 +1,3 @@
+import LanguageSettings from './settings-language.jsx';
```

Comment:
Why does the file name `settings-language` but imported as `LanguageSettings`?

Reply:
Changed